docs/content/concepts/automation/declarative-automation/customizing-automation-conditions.mdx

```diff
@@ -36,23 +32,24 @@ in_progress_or_failed_parents = AutomationCondition.any_deps_match(
-| `AutomationCondition.code_version_changed`  | The asset has a new code version since the previous evaluation                                                                    |
-| `AutomationCondition.cron_tick_passed`      | A new tick of the provided cron schedule occurred since the previous evaluation                                                   |
-| `AutomationCondition.in_latest_time_window` | The asset partition falls within the latest time window of the asset’s <PyObject object="PartitionsDefinition" />, if applicable. |
-| `AutomationCondition.will_be_requested`     | The asset partition will be requested in this tick                                                                                |
+Operands are base conditions which can be true or false about a given target. For partitioned assets, the target will be a given partition of the asset.
+
+| Operand                                     | Description                                                                                                          |
+| ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| `AutomationCondition.missing`               | Target has not been executed                                                                                         |
+| `AutomationCondition.in_progress`           | Target is part of an in-progress run                                                                                 |
+| `AutomationCondition.execution_failed`      | Target failed to be executed in its latest run                                                                       |
+| `AutomationCondition.newly_updated`         | Target was updated since the previous evaluation                                                                     |
+| `AutomationCondition.newly_requested`       | Target was requested on the previous evaluation                                                                      |
+| `AutomationCondition.code_version_changed`  | Target has a new code version since the previous evaluation                                                          |
+| `AutomationCondition.cron_tick_passed`      | A new tick of the provided cron schedule occurred since the previous evaluation                                      |
+| `AutomationCondition.in_latest_time_window` | Target falls within the latest time window of the asset’s <PyObject object="PartitionsDefinition" />, if applicable. |
+| `AutomationCondition.will_be_requested`     | Target will be requested in this tick                                                                                |
+| `AutomationCondition.initial_evaluation`    | This is the first evaluation of this condition                                                                       |
```

Comment:
As a follow up, a table of examples could go a long way of how these can be usefully combined to do specific use cases.